sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -10,9 +10,12 @@
     - Use `ServiceBusSender.schedule(messages, schedule_time_utc)` for scheduling messages.
     - Use `ServiceBusSender.cancel_scheduled_messages(sequence_numbers)` for scheduled messages cancellation.
 * `ServiceBusSender.send()` can now send a list of messages in one call, if they fit into a single batch.  If they do not fit a `ValueError` is thrown.
-* `BatchMessage.add()` and `ServiceBusSender.send()` raises `MessageContentTooLarge`, which is a subclass of `ValueError` if the content is over-sized.
-* `ServiceBusReceiver.receive()` raises `ValueError` if the max_batch_size is greater than the prefetch of `ServiceBusClient`.
+* `BatchMessage.add()` and `ServiceBusSender.send()` would raise `MessageContentTooLarge` if the content is over-sized.
```

Comment:
We noted the valueError subclass below, but we yanked it here?

Reply:
Yes this is a change so put it in the change log. `send()` used to raise `MessageSendFailed` if content is over-sized.

Reply:
Ah, I see how you're thinking about this now.  Sure, seems reasonable enough.